sound/soc/sof/ipc4-topology.c

```diff
@@ -2081,6 +2264,11 @@ static const struct sof_ipc_tplg_widget_ops tplg_ipc4_widget_ops[SND_SOC_DAPM_TY
 				src_token_list, ARRAY_SIZE(src_token_list),
 				NULL, sof_ipc4_prepare_src_module,
 				NULL},
+	[snd_soc_dapm_effect] = {sof_ipc4_widget_setup_comp_process,
```

Comment:
So, this patch add snd_soc_dapm_effect support for IPC4.
It is a detail that the IPC4 terminology is using 'process' for everything which does not have a clear name?

Reply:
Yes, this is what is done in IPC3. I'm not sure why we do it like this.

Reply:
What I meant is that the commit title should (imho) tell that the patch implements the effect support and the commit message details that it is internally done with 'process' things.

Reply:
Got it.